docs/release-guide/README.md

```diff
@@ -3,13 +3,14 @@
 This is the process we follow when releasing a new version of Cesium for Omniverse on GitHub.
 
 1. [Release a new version of Cesium for Omniverse Samples](#releasing-a-new-version-of-cesium-for-omniverse-samples).
+2. Update [`extension.py`](../../apps/exts/cesium.performance.app/cesium/performance/app/extension.py) in the Performance App with the latest public `ION_ACCESS_TOKEN`.
 3. Make sure the latest commit in `main` is passing CI.
 4. Download the latest build from S3. In the AWS management console (InternalServices AWS account), go to the bucket [`cesium-builds/cesium-omniverse/main`](https://s3.console.aws.amazon.com/s3/buckets/cesium-builds?region=us-east-1&prefix=cesium-omniverse/main/&showversions=false), find the appropriate date and commit hash to download the CentOS and Windows zip files (e.g. `CesiumGS-cesium-omniverse-linux-x86_64-xxxxxxx.zip` and `CesiumGS-cesium-omniverse-windows-x86_64-xxxxxxx.zip`)
 5. Verify that the Linux package loads in USD Composer (see instructions below).
 6. Verify that the Windows package loads in USD Composer (see instructions below).
 7. Update the project `VERSION` in [CMakeLists.txt](../../CMakeLists.txt).
 8. Update the extension `version` in [extension.toml](../../exts/cesium.omniverse/config/extension.toml). This should be the same version as above.
```

Comment:
@weegeekps it looks like the version numbers weren't updated in this PR. We might need to do a retag / re-release.